tests/Forms/PresentedOptionsFieldTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace SilverStripe\CKANRegistry\Tests\Forms;
+
+use SilverStripe\CKANRegistry\Forms\PresentedOptionsField;
+use SilverStripe\Dev\SapphireTest;
+
+class PresentedOptionsFieldTest extends SapphireTest
+{
+    public function testConstruct()
+    {
+        $field = new PresentedOptionsField('my-field');
```

Comment:
```suggestion
        $field = new PresentedOptionsField('my-field', new Resource);
```

^ or a mocked Resource object to pass through